RemoteHealthcare/HRManager.cs

```diff
@@ -64,8 +57,6 @@ public async void MakeConnection(int dataBlocks)
                     }
                 }
             }
-
-            return;
         }
 
         private static void BleHeart_SubscriptionValueChanged(object sender, BLESubscriptionValueChangedEventArgs e)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#code-structure) voor fout in code structuur.